spec/spec_helper.rb

```diff
@@ -21,6 +21,9 @@
 require 'codeclimate-test-reporter'
 CodeClimate::TestReporter.start
 
+require 'webmock/rspec'
+WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
@chrisdolendo I got that line from Webmock instructions, it is sort of confusing how Webmock + VCR work together but my understanding is that, if you wrap a request w/ VCR then it knows "ok I'm allowed to go outside the network if a cassette needs to be recorded" otherwise that line is basically saying "throw an exception if you try to connect outside the network, but allow localhost" e.g. if one of your tests is hitting `localhost:3000/xyz` or something, then that is allowed.